app/views/bed_layouts/_destroy_location.html.haml

```diff
@@ -0,0 +1,31 @@
+%div{data: { barcodes: barcodes || "", location: location }}
+  %p{style: "background-color: #e8f5e9;color: green;padding: 10px; width: 80%"}
+    Click the Search button to view all labware available in the selected location.
+%div
+  %label{for: "location"} Source location
+  = text_field_tag 'location', location, :size => 80, :onkeydown => "if(event.key == 'Enter') { event.preventDefault(); window.dispatchSearchEvent();}"
+  = button_tag "Search", :type => "button", :id => "search_button", :onclick => "window.dispatchSearchEvent()"
+- if barcodes.present?
+  %div{style: "margin-top: 40px; text-align: left;"}
+    %p{style: "margin-left: 40px;color: green;background-color: #e8f5e9;padding: 10px; width: 80%"}
```

Comment:
Is this the same style for similar messages for the same look and feel?

Reply:
There is no specific style in this UI. I used it because it gives a positive emphasis on the message. I didn't use it on the labware destruction page you created. If it's okay with you, I can change it there to maintain uniformity in displaying these information messages.

Reply:
Yes, that would be great actually.